browser/ui/webui/settings/brave_privacy_handler.cc

```diff
@@ -102,6 +105,9 @@ void BravePrivacyHandler::AddLoadTimeData(content::WebUIDataSource* data_source,
       "isOpenAIChatFromBraveSearchEnabled",
       ai_chat::IsAIChatEnabled(profile->GetPrefs()) &&
           ai_chat::features::IsOpenAIChatFromBraveSearchEnabled());
+  data_source->AddBoolean(
+      "isEmailAliasesFeatureEnabled",
```

Comment:
BravePrivacyHandler doesn't seem like the right place for this, @petemill @fallaciousreasoning ?

Reply:
I guess I could see the case being made that this belongs in here (it is a privacy feature), but it displays on the Autofill page, so maybe we should just add this in the root handler @arthuredelstein?

`brave/browser/ui/webui/brave_settings_ui.cc::AddResources`

Reply:
moved to the root handler